commands.go

```diff
@@ -54,6 +50,61 @@ func formatSec(dur time.Duration) string {
 	return strconv.FormatInt(int64(dur/time.Second), 10)
 }
 
+func stringify(vv interface{}) (string, error) {
+	switch v := vv.(type) {
+	case nil:
+		return "", nil
+	case string:
+		return v, nil
+	case []byte:
+		return string(v), nil
+	case int:
+		return formatInt(int64(v)), nil
+	case int8:
+		return formatInt(int64(v)), nil
+	case int16:
+		return formatInt(int64(v)), nil
+	case int32:
+		return formatInt(int64(v)), nil
+	case int64:
+		return formatInt(v), nil
+	case uint:
+		return formatUint(uint64(v)), nil
+	case uint8:
+		return formatUint(uint64(v)), nil
+	case uint16:
+		return formatUint(uint64(v)), nil
+	case uint32:
+		return formatUint(uint64(v)), nil
+	case uint64:
+		return formatUint(v), nil
+	case float32:
+		return formatFloat(float64(v)), nil
+	case float64:
+		return formatFloat(v), nil
+	case bool:
+		if v {
+			return "1", nil
+		} else {
+			return "0", nil
+		}
+	default:
+		b, err := Marshal(vv)
```

Comment:
I would probably define a `type Encodable interface { EncodeRedis() (string, error) }` and return an error on `default`